docs/installing_deis/index.rst

```diff
@@ -22,13 +22,14 @@ information on how to get set up with CoreOS.
     quick-start
     system-requirements
     aws
-    azure
+    baremetal
     digitalocean
+    engineyard
     gce
+    azure
```

Comment:
I had to build this PR to see how this alphabetizes things correctly: "Microsoft Azure"